main/src/ca/uwaterloo/flix/language/ast/ParsedAst.scala

```diff
@@ -1107,6 +1107,10 @@ object ParsedAst {
 
     case class Array(sp1: SourcePosition, elms: Seq[ParsedAst.Pattern], sp2: SourcePosition) extends ParsedAst.Pattern
 
+    case class ArrayTailSpread(sp1: SourcePosition, elms: Seq[ParsedAst.Pattern], sp2: SourcePosition, ident: Name.Ident, sp3: SourcePosition) extends ParsedAst.Pattern
```

Comment:
I don't think you need three SPs normally we have one SP at the beginning of an expression and one at the end. We don't use them in the middle.